idgenerator/aws/aws_xray_idgenerator.go

```diff
@@ -0,0 +1,84 @@
+// Copyright The OpenTelemetry Authors
```

Comment:
I'm not sure about this, but I will make an `x-ray` directory just in case in the future there are other AWS Tracing ID generators. Thanks for the great suggestion!